cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -5,39 +5,80 @@ import * as createtx from '../../e2e/pages/create_tx.pages'
 const sendValue = 0.00002
 const currentNonce = 0
 
+function happyPathToStepTwo() {
+  createtx.typeRecipientAddress(constants.EOA)
+  createtx.clickOnTokenselectorAndSelectSepoliaEth()
+  createtx.setSendValue(sendValue)
+  createtx.clickOnNextBtn()
+}
+
 describe('[SMOKE] Create transactions tests', () => {
-  before(() => {
+  beforeEach(() => {
     cy.clearLocalStorage()
     cy.visit(constants.BALANCE_URL + constants.SEPOLIA_TEST_SAFE_7)
     main.acceptCookies()
-  })
-
-  // TODO: Change title to: Verify a new send token transaction can be initiated
-  it('[SMOKE] Verify a new send token transaction can be created', () => {
     createtx.clickOnNewtransactionBtn()
     createtx.clickOnSendTokensBtn()
-    createtx.typeRecipientAddress(constants.EOA)
+  })
+
+  it('[SMOKE] Verify address input error values', () => {
+    createtx.verifyRandomStringAddress('Lorem Ipsum')
+    createtx.verifyWrongChecksum(constants.WRONGLY_CHECKSUMMED_ADDRESS)
+  })
+
+  it('[SMOKE] Verify address input validates a valid ENS name', () => {
```

Comment:
The test does not seem to test what it is expected to. Please review it and finalise accordingly.